app/Http/Controllers/Service/CentreController.php

```diff
@@ -34,4 +35,154 @@ public function printRegistrations(Centre $centre)
             ]
         );
     }
+
+    /**
+     * Exports a summary of registrations from the User's relevant Centres.
+     *
+     */
+    public function exportRegistrationsSummary()
+    {
+        // Get User
+        $user = Auth::user();
+
+        // Get now()
+        $now = Carbon::now();
+
+        // Get centres
+        $centres = $user->relevantCentres();
+
+        // get registrations and order them
+        $registrations = Registration::whereIn('centre_id', $centres->pluck('id')->all())
+            ->with(['centre','family.children','family.carers'])
+            ->orderBy('centre_id', 'asc')
+            ->get();
+
+        // set blank rows for laravel-excel
+        $rows = [];
+
+        // Looks like larevel-excel can't auto-generate headers
+        // by collating all the row keys and normalising.
+        // So we have to do it by hand.
+
+        // create base headers
+        $headers = [
+        ];
+
+        // Per registration...
+        foreach ($registrations as $reg) {
+            $row = [
+                "RVID" => $reg->family->rvid,
+                "Centre" => $reg->centre->name,
+                "Primary Carer" => $reg->family->carers->first()->name,
+                "Food Chart Received" => (!is_null($reg->fm_chart_on)) ? $reg->fm_chart_on->format('d/m/Y') : null,
+                "Food Diary Received" => (!is_null($reg->fm_diary_on)) ? $reg->fm_diary_on->format('d/m/Y') : null,
+                "Entitlement" => $reg->family->entitlement,
+            ];
+
+            // Per child dependent things
+            $kids = [];
+            $due_date = null;
+            $eligible = 0;
+            foreach ($reg->family->children as $index => $child) {
+                // make a 'Child X DoB' key
+                $child_index = $index + 1;
+                $status = $child->getStatus();
+                $dob_header = 'Child ' . (string)$child_index . ' DoB';
+
+                // Arrange kids by eligibility
+                switch ($status['eligibility']) {
+                    case 'Pregnancy':
+                        $due_date = $child->dob->format('d/m/Y');
+                        break;
+                    case 'Eligible':
+                        $kids[$dob_header] = $child->dob->format('m/Y');
+                        $eligible += 1;
+                        break;
+                    case "Ineligible":
+                        $kids[$dob_header] = $child->dob->format('m/Y');
+                        break;
+                }
+            }
+            // Add count oif eligible kids
+            $row["Eligible Children"] = $eligible;
```

Comment:
Spaces in the keys makes me sad - but I get that you are short cutting.

Reply:
Ditto above.